packages/bierzo-wallet/src/logic/connection.ts

```diff
@@ -11,20 +11,23 @@ let liskConnection: LiskConnection | undefined;
 
 async function getEthereumConnection(url: string, scraperApiUrl?: string): Promise<EthereumConnection> {
   if (!ethereumConnection) {
+    // eslint-disable-next-line require-atomic-updates
     ethereumConnection = await EthereumConnection.establish(url, { scraperApiUrl });
   }
   return ethereumConnection;
 }
 
 async function getBnsConnection(url: string): Promise<BnsConnection> {
   if (!bnsConnection) {
+    // eslint-disable-next-line require-atomic-updates
     bnsConnection = await BnsConnection.establish(url);
   }
   return bnsConnection;
 }
 
 async function getLiskConnection(url: string): Promise<LiskConnection> {
   if (!liskConnection) {
+    // eslint-disable-next-line require-atomic-updates
```

Comment:
👍 I tried to understand those warnings but gave up after some time. I don't see how this can be problematic in our use case

Reply:
Me too